server/ntb/io/feed_parsers/ntb_tt_ninjs.py

```diff
@@ -0,0 +1,60 @@
+# -*- coding: utf-8; -*-
+#
+# This file is part of Superdesk.
+#
+# Copyright 2013, 2014 Sourcefabric z.u. and contributors.
+#
+# For the full copyright and license information, please see the
+# AUTHORS and LICENSE files distributed with this source code, or
+# at https://www.sourcefabric.org/superdesk/license
+
+from superdesk.io.feed_parsers.ninjs import NINJSFeedParser
+from superdesk.io.registry import register_feed_parser
+from superdesk.utc import utc
+from dateutil.parser import parse
+import json
+
+
+class NTBTTNINJSFeedParser(NINJSFeedParser):
+    """
+    Feed Parser for NTB TT NINJS format
+    """
+
+    NAME = 'ntb_tt_ninjs'
+
+    label = 'NTB TT NINJS Feed Parser'
+
+    def __init__(self):
+        super().__init__()
+
+    def parse(self, file_path, provider=None):
```

Comment:
I'm just wondering why overriding this parse method so you have to deal with json loading instead of doing the custom bits in `transform_from_ninjs`:
```py
def _transform_to_ninjs(ninjs):
    item = super()._transform_to_ninjs(ninjs)
    return self._transform_to_ntb_tt_ninjs(item)
```

Reply:
you mean, moving the transform calls to a different method?